compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/optimizer/LargeMethodOptimizer.java

```diff
@@ -710,7 +710,7 @@ private BIRBasicBlock generateSplitsInSameBB(BIRFunction function, int bbNum, Li
      * @return The generated operand for the variable declaration
      */
     private BIROperand generateTempLocalVariable(BIRFunction func, BType variableType) {
-        Name variableName = new Name("$split$tempVar$" + splitTempVarNum++);
+        Name variableName = new Name("$split$tempVar$_" + splitTempVarNum++);
```

Comment:
Done for clarity, as suggested by the PR review comment https://github.com/ballerina-platform/ballerina-lang/pull/34197#discussion_r767459279.